static/app/views/explore/logs/logsIssuesSection.tsx

```diff
@@ -0,0 +1,69 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import Feature from 'sentry/components/acl/feature';
+import {SearchQueryBuilder} from 'sentry/components/searchQueryBuilder';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {
+  LogsPageParamsProvider,
+  type LogsPageParamsProviderProps,
+  useLogsSearch,
+  useSetLogsQuery,
+} from 'sentry/views/explore/contexts/logs/logsPageParams';
+import {LogsTable} from 'sentry/views/explore/logs/logsTable';
+import {useExploreLogsTable} from 'sentry/views/explore/logs/useLogsQuery';
+import {SectionKey} from 'sentry/views/issueDetails/streamline/context';
+import {InterimSection} from 'sentry/views/issueDetails/streamline/interimSection';
+
+export function LogsIssuesSection({
+  initialCollapse,
+  isIssuesDetailView,
+  limitToTraceId: traceId,
+}: {
+  initialCollapse: boolean;
+} & Omit<LogsPageParamsProviderProps, 'children'>) {
+  return (
+    <Feature features={['ourlogs-enabled']}>
+      <InterimSection
```

Comment:
the section shouldn't render if there are no results

Reply:
sure